easybuild/easyconfigs/p/pbcopper/pbcopper-2.0.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'MesonNinja'
+
+name = 'pbcopper'
+version = '2.0.0'
+
+homepage = 'https://github.com/PacificBiosciences/pbcopper'
+description = """The pbcopper library provides a suite of data structures, algorithms,
+ and utilities for C++ applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

since we're depending on a `GCC`-level easyconfig, this should also be (at least) at `GCC` level. Note that the filename will need updating to match
```suggestion
toolchain = {'name': 'GCC', 'version': '11.2.0'}
```